java/ql/lib/change-notes/2022-12-09-default-extension-methods.md

```diff
@@ -0,0 +1,4 @@
+---
+category: minorAnalysis
+---
+* The extraction of extension methods have been improved when default parameter values are present. The dispatch and extension receiver parameters are extracted in the correct order. The `ExtensionMethod::getExtensionParameterIndex` predicate has been introduced to facilitate getting the correct extension parameter index.
```

Comment:
```suggestion
* The extraction of extension methods has been improved when default parameter values are present. The dispatch and extension receiver parameters are extracted in the correct order. The `ExtensionMethod::getExtensionParameterIndex` predicate has been introduced to facilitate getting the correct extension parameter index.
```

Also, should we mention Kotlin somewhere here?

Reply:
Thanks. Let me adjust it.